use-cases/predictive_maintenance/1_dataprep_dw_job_predmaint.ipynb

```diff
@@ -4,18 +4,45 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    output_name = "ff586e7b-a02d-472b-91d4-da3dd05d7a30.default"
</pre><p>Why is the output name this oddly specific string? Can it be more descriptive?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Oh yes, this oddly specific string is a reference to the DataWranglerflow file - it uses this string to identify the methods in the flow file. I can add an explanation</p>

Reply:
<p>Added more details - let me know what you think</p>